frontend/src/components/menu/useMenu.tsx

```diff
@@ -0,0 +1,44 @@
+import defaultUserAvatar from "assets/defaultUserAvatar.png";
+import { MenuUserImage, MenuUserName } from "components/menu/Menu.styled";
+import dependlyComponentDisplay from "helpers/dependly-component-display";
+import { useState } from "react";
+import useCloseByEsc from "hooks/useCloseByEsc";
+import useFetchAvatar from "hooks/useFetchAvatar";
+import useFetchUser from "hooks/useFetchUser";
+
+export const useMenu = () => {
+    const [showMenu, setShowMenu] = useState(false);
+
+    const { username, userAvatar } = useFetchUser();
+    const { userAvatarUrl, error, isLoading } = useFetchAvatar(userAvatar?.id);
+
+    const changeMenuVisible = () => {
+        setShowMenu(prevState => !prevState);
+    };
+
+    const menuProfileHandler = () => {
```

Comment:
maybe rename to "menuProfileContentDisplay"

Reply:
Not that way, back old name to this const inside, i leave this comment in line 19, i want to change name of that "...Handler"